ops/roles/infra/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Install common required packages
+  apt:
+    pkg:
+      - build-essential
+      - curl
+      - git
+      - zip
```

Comment:
Why are those necessary? 🤔

Reply:
`git` in order to `git clone` the repository

`zip` is not, I remove it